_posts/2015-11-16-grace-hopper-tips.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Grace Hopper Celebration Tips
+tags: ghc grace hopper women 
+username: tash1207 
+---
+
```

Comment:
:sheep: Just a suggestion, but I feel like this post deserves a pic of you at the event. Would give a good sense of what it was like. You take any sweet selfies?